cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -233,6 +233,18 @@ parameters:
   value: ""
 - name: FPA_CERT_NAME
   description: The name of the secret that contains the first party application certificate bundle.
+- name: ARM_HELPER_IDENTITY_CLIENT_ID
+  description: The client id of the service principal that represents the ARM Helper Identity.
+  value: "57e54810-3138-4f38-bd3b-29cb33f4c358"
+- name: ARM_HELPER_IDENTITY_KEYVAULT_NAME
+  description: The name of the Key Vault containing keys, secrets and certificates used to support ARM Helper Indentity.
+  value: "aro-dev-first-party"
+- name: ARM_HELPER_IDENTITY_TENANT_ID
+  description: The Tenant ID of the service principal that represents the ARM Helper Identity.
+  value: "64dc69e4-d083-49fc-9569-ebece1dd1408"
+- name: ARM_HELPER_IDENTITY_CERT_NAME
+  description: The name of the secret that contains the ARM Helper Indentity certificate bundle.
+  value: "d3dea586-11bb-428a-bc27-465a687e88cc"
```

```suggestion
  value: ""
```
As FPA you can add this as a variable, it is specified in the Makefile. Also it should be a name and not the id.